main.py

```diff
@@ -1,10 +1,47 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player_info in players.items():
+        race = player_info.get("race")
+        guild = player_info.get("guild")
+        skills = race.get("skills")
+
+        if not Race.objects.filter(name=race["name"]).exists():
+            current_race = Race.objects.create(
+                name=race["name"],
+                description=race["description"]
+            )
+
+            for skill in skills:
+                if not Skill.objects.filter(name=skill["name"]).exists():
+                    Skill.objects.create(
+                        name=skill["name"],
+                        bonus=skill["bonus"],
+                        race_id=current_race.id
+                    )
+        if guild:
+            if not Guild.objects.filter(name=guild["name"]).exists():
+                Guild.objects.create(
+                    name=guild["name"],
+                    description=guild["description"]
+                )
+            guild = Guild.objects.get(name=guild["name"])
```

Comment:
you can use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create
The same with other places